src/fondant/pipeline/runner.py

```diff
@@ -126,3 +126,32 @@ def get_name_from_spec(self, input_spec: str):
         with open(input_spec) as f:
             spec = yaml.safe_load(f)
             return spec["pipelineInfo"]["name"]
+
+
+class SagemakerRunner(Runner):
+    def __init__(self):
+        self.__resolve_imports()
+
+    def __resolve_imports(self):
+        import boto3
+
+        self.boto3 = boto3
+
+    def run(self, input_spec: str, *args, **kwargs):
+        """Run a sagemaker pipeline."""
+        client = self.boto3.client("sagemaker")
+
+        with open(input_spec) as f:
+            pipeline = f.read()
+            response = client.create_pipeline(
+                PipelineName="my-pipeline4",
+                PipelineDefinition=pipeline,
+                RoleArn="TODO",
+            )
+            print(response)
```

Comment:
Probably for debugging, let's switch it to `logging` (debug or info depending on what you think is more suitable)